servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -3773,9 +3786,28 @@ void RendererSceneRenderRD::render_scene(RID p_render_buffers, const CameraData
 	_render_scene(&render_data, clear_color);
 
 	if (p_render_buffers.is_valid()) {
-		if (debug_draw == RS::VIEWPORT_DEBUG_DRAW_CLUSTER_OMNI_LIGHTS || debug_draw == RS::VIEWPORT_DEBUG_DRAW_CLUSTER_SPOT_LIGHTS || debug_draw == RS::VIEWPORT_DEBUG_DRAW_CLUSTER_DECALS || debug_draw == RS::VIEWPORT_DEBUG_DRAW_CLUSTER_REFLECTION_PROBES) {
+		/*
```

Comment:
Does not look right

Reply:
@reduz I had to move this into the clustered renderer as we do the final stage differently for the mobile renderer now. I just haven't removed the old code yet.